src/main/java/Test.java

```diff
@@ -0,0 +1,34 @@
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
```

Comment:
Please use the correct header and author as we discussed

Reply:
Removed this file